packages/orbit-components/src/Wizard/WizardStep/index.js

```diff
@@ -0,0 +1,246 @@
+// @flow
+import * as React from "react";
+import { css } from "styled-components";
+import { hex2rgb } from "@csstools/convert-colors";
+
+import useMediaQuery from "../../hooks/useMediaQuery";
+import ButtonLink from "../../ButtonLink";
+import Stack from "../../Stack";
+import Text from "../../Text";
+import CheckCircle from "../../icons/CheckCircle";
+import useTheme from "../../hooks/useTheme";
+
+import type { Props } from ".";
+
+const WizardStep = ({
+  dataTest,
+  number,
+  title,
+  status,
+  nextStepStatus,
+  active,
+  onClick,
+}: Props) => {
+  const { isTablet } = useMediaQuery();
+  const theme = useTheme();
+  const iconWidth = parseFloat(theme.orbit.widthIconSmall);
+  const iconHeight = parseFloat(theme.orbit.heightIconSmall);
+  // account for additional inner spacing because of visual balance
+  const checkboxWidth = iconWidth * (5 / 6);
+  const checkboxHeight = iconHeight * (5 / 6);
+  const colorDisabled = theme.orbit.paletteCloudNormalHover;
+
+  const activeGlowColor = `rgba(${hex2rgb(theme.orbit.paletteProductNormal)
+    .slice(0, -1)
+    .map(n => 255 * (n / 100))
+    .concat(0.2)
+    .join(", ")})`;
+
+  /**
+   * We're explicitly rounding values in layout styles because browsers seem to
+   * be rounding them inconsistently horizontally vs vertically.
+   */
+
+  const icon = (
```

Comment:
Consider to make it as separate component inside WizardStep :) 

I think it's better to split it here, less lines of code, better reading and later code maintain etc.

Reply:
Good idea! This is a bit awkward.